poetry/orb.yaml

```diff
@@ -26,16 +26,24 @@ commands:
         type: string
     steps:
       - run:
-          name: install curl
+          name: Install curl
           command: |
-            if which apk >/dev/null; then
-              apk add --no-cache --no-progress curl
-            elif which apt-get >/dev/null; then
-              apt-get update -qy
-              apt-get install -qy curl
+            if cat /etc/issue | grep "Alpine" >/dev/null 2>&1; then
+              if [ "$ID" = 0 ]; then export SUDO=""; else export SUDO="sudo"; fi
             else
-              echo >&2 "ERROR: could not find supported package manager"
-              exit 1
+              if [[ $EUID == 0 ]]; then export SUDO=""; else export SUDO="sudo"; fi
+            fi
+            echo "export SUDO=$SUDO" >> ${BASH_ENV}
```

Comment:
I don't think we need this -- the `SUDO` variable isn't used in any future steps here, so we can keep it contained into only this step rather than potentially affecting future tasks.